corehq/apps/export/views/list.py

```diff
@@ -851,3 +856,54 @@ def submit_app_data_drilldown_form(request, domain):
         'success': True,
         'url': reverse(cls.urlname, args=[domain]) + url_params,
     })
+
+
+class ODataFeedListHelper(ExportListHelper):
+    allow_bulk_export = False
+    form_or_case = 'case'
+    is_deid = False
+    include_saved_filters = True
+
+    @property
+    def create_export_form_title(self):
+        return _("Select a model to export")
+
+    @memoized
+    def get_saved_exports(self):
+        if self.permissions.has_case_export_permissions:
+            exports = get_case_exports_by_domain(self.domain,
+                                                 self.permissions.has_deid_view_permissions,
+                                                 include_docs=False)
+        return [x for x in exports if x['is_odata_config']]
```

Comment:
The way it's currently structured, there are a bunch of different "Export Lists", corresponding to what you see on the sidebar:

![image](https://user-images.githubusercontent.com/2367539/59366939-1fd84500-8d09-11e9-81d7-93a95a52fdea.png)

Each of these has a "Helper" class which contains logic specific to that list.  One of these bits of logic the helper must implement is `get_saved_exports`:
https://github.com/dimagi/commcare-hq/blob/acd5f7c925e6508dab376c3a248a89118dd2e027/corehq/apps/export/views/list.py#L152-L157
This is the thing responsible for fetching the master list of saved export configs which should appear on that list page.

I do think it makes sense for this to live on the helper class, since that's the home for list-specific logic, though I would like a better way to decide which goes where.

One option would be to invert it - have a function which takes an export stub and returns an Enum corresponding to the appropriate helper, a la [ExportListHelper.get](https://github.com/dimagi/commcare-hq/blob/53c5c0cb9308069fb50f26750982fbc646e6750f/corehq/apps/export/views/list.py#L71-L93).  That would ensure that a given config appears in only one place, but does sound more convoluted, and we'd still probably have to handle DeID separately. Really what I want is for there to be one definitive field on the model describing where a config lives, though I'm not sure it's that simple.